apps/web/src/theme/index.tsx

```diff
@@ -143,9 +143,7 @@ function applyOverriddenColors(defaultColors: ThemeColors, overriddenColors?: Pa
 }
 
 export function ThemeProvider({ children, ...overriddenColors }: PropsWithChildren<Partial<ThemeColors>>) {
-  //const darkMode = useIsDarkMode()
-  // TODO: Enable when dark mode is completed ref: https://github.com/hemilabs/interface/issues/16
-  const darkMode = false
+  const darkMode = useIsDarkMode()
```

Comment:
Isn't it simpler to just do this and remove the hook?

```suggestion
  const darkMode = false
```

Reply:
I'd have to do it everywhere it's called (many places), and then put it back once we re-enable it.